packages/aiqtoolkit_weave/src/aiq/plugins/weave/weave_sdk.py

```diff
@@ -26,19 +26,55 @@ class WeaveTelemetryExporter(TelemetryExporterBaseConfig, name="weave"):
     """A telemetry exporter to transmit traces to Weights & Biases Weave using OpenTelemetry."""
     project: str = Field(description="The W&B project name.")
     entity: Optional[str] = Field(default=None, description="The W&B username or team name.")
+    redact_pii: bool = Field(default=False, description="Whether to redact PII from the traces.")
+    redact_pii_fields: Optional[List[str]] = Field(
+        default=None,
+        description="Custom list of PII entity types to redact. Only used when redact_pii=True. "
+                   "Examples: CREDIT_CARD, EMAIL_ADDRESS, PHONE_NUMBER, etc."
+    )
+    redact_keys: Optional[List[str]] = Field(
+        default=None,
+        description="Additional keys to redact from traces beyond the default (api_key, auth_headers, authorization)."
+    )
```

Comment:
```suggestion
    redact_pii_fields: Optional[list[str]] = Field(
        default=None,
        description="Custom list of PII entity types to redact. Only used when redact_pii=True. "
                   "Examples: CREDIT_CARD, EMAIL_ADDRESS, PHONE_NUMBER, etc."
    )
    redact_keys: Optional[list[str]] = Field(
        default=None,
        description="Additional keys to redact from traces beyond the default (api_key, auth_headers, authorization)."
    )
```
These are deprecated